src/stdio_mgr/stdio_mgr.py

```diff
@@ -95,62 +101,32 @@ def getvalue(self):
             return self._stream.getvalue().decode(self.encoding)
 
 
-@attr.s(slots=False)
-class TeeStdin(TextIOWrapper):
+class Tee(TextIOWrapper):
     """Class to tee contents to a side buffer on read.
 
     Subclass of :class:`~io.TextIOWrapper` that overrides
     :meth:`~io.TextIOBase.read` and :meth:`~io.TextIOBase.readline`
-    to tee all content *read* from the stream to `tee`. The
-    canonical use-case is with :func:`stdio_mgr`,
-    where `tee` is the mocked stream for `stdin`.
+    to tee all content *read* from the stream to `tee`.
 
     To emphasize: teeing occurs on content *read*, **not write**.
 
     As a subclass of :class:`~io.TextIOWrapper`, it is not thread-safe.
 
-    This class provides :meth:`~TeeStdin.getvalue` which emulates the
-    behavior of :meth:`StringIO.getvalue() <io.StringIO.getvalue>`,
-    decoding the buffer using the :attr:`~io.TextIOBase.encoding`.
-
-    This class also provides the method
-    :meth:`~TeeStdin.append`, which is not available
-    for the base :class:`TextIOWrapper <io.TextIOBase>` type.
-    This method adds new content to the end of the
-    stream while leaving the read position unchanged.
-
-    Instantiation takes two arguments:
+    Instantiation takes an additional argument:
 
     `tee`
 
         :class:`~io.TextIOBase` -- Text stream to receive
         content teed from :class:`TeeStdin` upon read
```

Comment:
This is no longer `TeeStdin`; it's just `Tee`, as things sit currently.

Reply:
The mention of TeeStdin feels a bit unnecessary here, as it is implicit that any other instance would be receiving from this class during methods of this class.